Modules/HR/Config/config.php

```diff
@@ -26,6 +26,12 @@
         'closed' => 'Closed'
     ],
 
+    'domain-status' => [
```

Comment:
Let's keep the naming convention consistent:
```suggestion
    'domains' => [
```

Reply:
Also, since this is domain for an opportunity and not a generic domain, we should keep it inside opportunities:
```php
'opportunities' => [
    'domains' => [
        ....
```

and then use it like:
```php
config("hr.opportunities.domains")
```